dv/parameters/KDV01_CubeOrangePlusDefault.param

```diff
@@ -832,13 +897,13 @@ SERIAL0_PROTOCOL,2.0
 SERIAL1_BAUD,57.0
 SERIAL1_OPTIONS,0.0
 SERIAL1_PROTOCOL,2.0
-SERIAL2_BAUD,57.0
+SERIAL2_BAUD,115.0
 SERIAL2_OPTIONS,0.0
 SERIAL2_PROTOCOL,2.0
-SERIAL3_BAUD,230.0
+SERIAL3_BAUD,38.0
 SERIAL3_OPTIONS,0.0
 SERIAL3_PROTOCOL,5.0
-SERIAL4_BAUD,230.0
+SERIAL4_BAUD,38.0
 SERIAL4_OPTIONS,0.0
 SERIAL4_PROTOCOL,5.0
 SERIAL5_BAUD,57.0
```

Comment:
Should be 115

Reply:
changed all serial ports to 115200. Will test if all still works

Reply:
I don't know what state of mind I was in when commenting this. That comment was meant for serial1. We should use what is well tested for the GNSS modules, not another value based on my brain fart :)